site/src/components/Icons/VSCodeIcon.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { Story } from "@storybook/react"
+import { VSCodeIcon } from "./VSCodeIcon"
+
+export default {
+  title: "icons/VSCodeIcon",
+  component: VSCodeIcon,
+}
+
+const Template: Story = (args) => <VSCodeIcon {...args} />
+
+export const Example = Template.bind({})
+Example.args = {}
```

Comment:
I see this is a pattern we have but it strikes me as sort of funny to have icon stories divorced from a page context. Where are these icons sourced from?

Reply:
I continued this pattern and didn't look too much into the why here. It is odd to have them disconnected from a page, and since they're presumably already on pages it might make sense just to remove these stories.

Reply:
Agree as stories are sort of intensive to generate. I will remove them in a separate PR; you don't have to change anything here.

Reply:
Ahh neat, thanks Kira!